Src/Campus.Master.API/Startup.cs

```diff
@@ -31,25 +32,35 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            var xmlDocFile = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
-            var xmlDocPath = Path.Combine(AppContext.BaseDirectory, xmlDocFile);
-            
+            string xmlDocFile = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
+            string xmlDocPath = Path.Combine(AppContext.BaseDirectory, xmlDocFile);
+
             services.AddControllers();
             services.AddCors();
             services.AddSwaggerGen(c =>
             {
-                c.SwaggerDoc(ApiVersion, new OpenApiInfo { Title = ApiTitle, Version = ApiVersion });
-                
+                c.SwaggerDoc(ApiVersion, new OpenApiInfo {Title = ApiTitle, Version = ApiVersion});
+
                 c.AddSecurityDefinition("Bearer",
-                    new OpenApiSecurityScheme { In = ParameterLocation.Header,
-                        Description = "Please enter into field the word 'Bearer' following by space and JWT", 
-                        Name = "Authorization", Type = SecuritySchemeType.ApiKey });
-                c.AddSecurityRequirement(new OpenApiSecurityRequirement {
-                    { new OpenApiSecurityScheme {Name = "Bearer"}, Enumerable.Empty<string>().ToList() },
+                    new OpenApiSecurityScheme
+                    {
+                        In = ParameterLocation.Header,
+                        Description = "Please enter into field the word 'Bearer' following by space and JWT",
+                        Name = "Authorization", Type = SecuritySchemeType.ApiKey
+                    });
+                c.AddSecurityRequirement(new OpenApiSecurityRequirement
+                {
+                    {new OpenApiSecurityScheme {Name = "Bearer"}, Enumerable.Empty<string>().ToList()},
                 });
-                
+
                 c.IncludeXmlComments(xmlDocPath);
             });
+            
+            services.AddScoped<IDbConnection>(c => new SqlConnection(Configuration["ConnectionStrings:Default"]));
```

Comment:
You shouldn't register service explicitly like this, because you're dragging dependencies of lower layers to the higher. Write service providing extension methods for the Data and Business layers. Basically, we'll get rid of unused dependencies in the API layer like SQL server dependencies. Check out extension method example here: https://github.com/antonAce/onion-di/blob/master/src/OnionDI/OnionDI.Data/Dependencies/DataServicesExtensions.cs

Reply:
Done